libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapSettings.java

```diff
@@ -0,0 +1,106 @@
+package com.mapbox.services.android.navigation.ui.v5.map;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.mapbox.services.android.navigation.ui.v5.camera.NavigationCamera;
+
+import static com.mapbox.services.android.navigation.ui.v5.map.MapFpsDelegate.DEFAULT_MAX_FPS_THRESHOLD;
+
+class NavigationMapSettings implements Parcelable {
```

Comment:
Any reason this class isn't immutable with a builder?

Reply:
@devotaaabel yeah it needs to be able to update as the `public` APIs in `NavigationMapboxMap` push values to it.  This object serves as a "cache" for the way name / fps settings, holding those values until the objects are instantiated.